src/assets/main.css

```diff
@@ -1,35 +1,109 @@
 @import './base.css';
 
-#app {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+@font-face {
+  font-family: "Black";
```

Comment:
use different variant of front with same font-family but with different weight and style

Reply:
used only IRANSandWeb(FaNum)

Reply:
As I describe it in telegram, use default fontiran example